.renovaterc.json5

```
@@ -127,5 +127,11 @@
         "\"openApiSupportedVersion\": \"~(?<currentValue>[^\"]+)\""
       ]
     }
-  ]
+  ],
+  "github-actions": {
+    "fileMatch": [
+      "(^|/)(__dot__github|workflow-templates|\\.(?:github|gitea|forgejo)/(?:workflows|actions))/.+\\.ya?ml$",
+      "(^|/)action\\.ya?ml$"
```

the `fileMatch` rules are additive, no need to redeclare the default ones
```suggestion
      "/__dot__github/.*/.+\\.ya?ml$"
```
also, you'll need to remove `**/templates/**` from `ignorePaths` in this file to make it process the template files